examples/file_scan/file_scan_examples/file_scan_sync.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+import pangea.exceptions as pe
+from pangea.config import PangeaConfig
+from pangea.services import FileScan
+from pangea.tools import logger_set_pangea_config
+
+token = os.getenv("PANGEA_FILE_SCAN_TOKEN")
+domain = os.getenv("PANGEA_DOMAIN")
+# To work in sync it's need to set up queue_retry_enable to true and set up a proper timeout
+# If timeout it's so little service won't end up and will return an AcceptedRequestException anyway
+config = PangeaConfig(domain=domain, queued_retry_enabled=True, poll_result_timeout=120)
+client = FileScan(token, config=config, logger_name="pangea")
+logger_set_pangea_config(logger_name=client.logger.name)
+
+FILEPATH = "./file_scan_examples/testfile.pdf"
+
+
+def main():
+    print("Checking file...")
+
+    try:
+        with open(FILEPATH, "rb") as f:
+            response = client.file_scan(file=f, verbose=True, provider="crowdstrike")
```

Comment:
FileScan is a good example to show and tests 202 handler in SDKs. All SDKs have the option that in case of a 202 try to poll result for a while, until user timeout is reached. In this case we have 2 example due to, in one case we enable 202 poll result with a longer timeout, and in the other example we disable that and show SDK user how to do a request, got a 202, wait (doing whatever they want) and then poll the result.
sync_call is not documented, all 202 handled is up to the SDK settings, nothing in the backend.